config/users/user-model.js

```diff
@@ -0,0 +1,25 @@
+
+//IMPORTS ⬇︎
+const db = require( '../../database/dbConfig' );
+
+//EXPORTS ⬇︎
+module.exports = {
+    add,
+    find,
+    findBy,
+}
+
+function find() {
+    return db('users').select('id', 'username')
+};
+
+//LOGIN FUNCTIONS
+function findBy( filter ) {
+    return db( 'users' ).where( filter );
+};
+
+//REGISTER FUNCTIONS
+async function add( user ) {
+    const [ id ] = await db( 'users' ).insert( user );
```

Comment:
i notice you are using async on this one but used promises and the login method, i would pick one and use that in all the methods for consistency in the code